src/browser_action.js

```diff
@@ -425,3 +425,10 @@ browseraction.createEventDiv_ = function(event) {
 window.addEventListener('load', function() {
   browseraction.initialize();
 }, false);
+
+window.addEventListener('keydown', function(event) {
```

Comment:
Shouldn’t this event listener be added to the textarea only, not the entire page?